programming_examples/matrix_multiplication/bf16/run_makefile_peano.lit

```diff
@@ -0,0 +1,8 @@
+// (c) Copyright 2025 Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: MIT
+//
+// REQUIRES: ryzen_ai_npu1, peano
+//
+// RUN: make -f %S/Makefile clean
```

Comment:
This will collide with `run_makefile_chess.lit` in the build/test directory., it is necessary for each lit test to run in its own subdirectory.
for example,
```
// RUN: mkdir -p test
// RUN: cd test
```
Where 'test' is unique for each of the lit scripts in a directory.

Reply:
Thanks for spotting this. I missed this one when I updated all tests. This is now fixed.